### PR TITLE
migrate cronjob and hpa to new api versions

### PR DESCRIPTION
**Pull request by pauvos**
* Cronjob v1beta1 => v1
* HorizontalPodAutoscaler v2beta2 => v2

see https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25

### CODE REVIEW

**Comment by stakater-user** [comment]
@pauvos Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/application/commit/fe536f76c4f5dc84c65a16f9756084f18d452989/checks) has Failed!

**Comment by rasheedamir** [comment]
@pauvos can you plz update this?

**Review by d3adb5 — APPROVED** [review]

**Comment by d3adb5** [comment]
Looks good to me, but since I'm a new maintainer, I'd rather have another maintainer take a look and let me know if I missed something like a necessary addition to the CHANGELOG or something similar.

File: application/templates/hpa.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {{- if and (.Values.autoscaling).enabled .Values.deployment }}
+{{ if $.Capabilities.APIVersions.Has "autoscaling/v2/HorizontalPodAutoscaler" -}}
```
Comment (mustafaStakater) [inline application/templates/hpa.yaml:2]: Why use $ when youre not within `with` or `range`

**Review by mustafaStakater — APPROVED** [review]

**Pull request merged**